api/v1/endpoints/queries.sql

```diff
@@ -99,6 +99,7 @@ WITH object_data AS (
         aar.role,
         aar.seb_config,
         aar.show_closed_assessment,
+        aar.show_closed_assessment_grade,
```

Comment:
I'm sorry to ask this, but could we rename this to `show_closed_assessment_score`? Within PrairieLearn we have generally been using the term "score" as the noun for the numerical score on each question/assessment, and we use "grade" for the verb of the action that computes scores. I'm not going to attempt to justify this choice of terminology 😄

Reply:
Done!